locales/en-US/experiments.ftl

```diff
@@ -126,6 +137,16 @@ pulseContributors1Title = Firefox UX
 pulseContributors2Title = Firefox Platform Product Intelligence
 pulseContributors3Title = Firefox QA
 pulseEolwarning = The Pulse experiment is ending soon, but the data you provided helped us so much that we want to build the feature in Firefox. We will uninstall this experiment soon.
+sendDescription = Send files through a safe, private, and encrypted link that automatically expires to ensure your stuff does not remain online forever.
+sendIntroduction = TKTKTKTK
```

Comment:
FWIW, ["TK" is a common editing/publishing thing for "to come soon"](https://en.wikipedia.org/wiki/To_come_(publishing)). This PR is work-in-progress per the WIP in the title